geodata_mart/templates/components/_footer_default.html

```diff
@@ -1,8 +1,179 @@
+        font-family: sans-serif;
+    }
+
+    html, body {
+        height: 100%;
+    }
+
+    .container {
+        min-height: 100%;
+        position: relative;
+    }
+
+    footer {
+        position: fixed;
+        left: 0;
+        bottom: 0 !important;
+        width: 100%;
+        background: #484444;
+        color: #fff;
+        padding-top: 0px; /* Adjust the top padding here */
+        padding-bottom: 0px; /* Adjust the bottom padding here */
+        z-index: 0; /* Move the footer to the background */
+    }
+
+    .footer-content {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        flex-direction: column;
+        text-align: center;
+    }
+
+    .footer-content h3 {
+        font-size: 2.1rem;
+        font-weight: 500;
+        text-transform: capitalize;
+        line-height: 3rem;
+    }
+
+    .footer-content p {
+        max-width: 500px;
+        margin: 10px auto;
+        line-height: 28px;
+        font-size: 14px;
+        color: #cacdd2;
+    }
+
+    .socials {
+        list-style: none;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        margin: 1rem 0 3rem 0;
+    }
+
+    .socials li {
+        margin: 0 10px;
+    }
+
+    .socials a {
+        text-decoration: none;
+        color: #fff;
+        border: 1.1px solid white;
+        padding: 5px;
+        border-radius: 50%;
+    }
+
+    .socials a i {
+        font-size: 1.1rem;
+        width: 20px;
+        transition: color .4s ease;
+    }
+
+    .socials a:hover i {
+        color: aqua;
+    }
+
+    .footer-bottom {
+        position: absolute;
+        bottom: 0;
+        left: 0;
+        width: 100%;
+        background: #000;
+        padding: 20px;
+        padding-top: 0px;
+        padding-bottom: 0px; /* Adjust the height here */
+        text-align: center;
+    }
+
+    .footer-bottom p {
+        float: right; /* Adjust the position of text here here */
+        font-size: 14px;
+        word-spacing: 2px;
+        text-transform: capitalize;
+    }
+
+    .footer-bottom p a {
+        color: #44bae8;
+        font-size: 16px;
+        text-decoration: none;
+    }
+
+    .footer-bottom span {
+        text-transform: uppercase;
+        opacity: .4;
+        font-weight: 200;
+    }
+
+    .footer-menu {
+        float: left;
+    }
+
+    .footer-menu ul {
+        display: flex;
+    }
+
+    .footer-menu ul li {
+        padding-right: 10px;
+        display: block;
+    }
+
+    .footer-menu ul li a {
+        color: #cfd2d6;
+        text-decoration: none;
+    }
+
+    .footer-menu ul li a:hover {
+        color: #27bcda;
+    }
+
+    @media (max-width: 500px) {
+        .footer-menu ul {
+            display: flex;
+            margin-top: 10px;
+            margin-bottom: 20px;
+        }
+    }
+</style>
+<footer class="text-secondary">
+    <div class="footer-content">
+        <h3>Spatial Data Downloader</h3>
```

Comment:
@NyakudyaA I have put a comment here #101. The name of the site will be Geospatial Data Downloader.